### DISCUSSION TITLE
Easy/beginner friendly issues?

### DISCUSSION THREAD

**Discussion by maxking** [kubernetes-el/kubernetes-el#269] [General]
It would be good to have some issues marked as easy or beginner friendly and/or have some new issues that can be a good point for a potential new contributor to get started with. I am looking into contributing to the project, and it seems like there is a good list of functionality to contribute to, but something easy to familiarize with the project would be great! :-)

**Comment by jinnovation** [comment]
Hi hi, thanks for the interest 👍 labels such as `good first issue` would be a fantastic idea; I'll see if I can ID some good candidates (@noorul, feel free to do the same as well).

Let's also use this thread to collect some general "starter tasks" that might not be amenable to specific, completable issues just yet.

**Reply by jinnovation** [reply]
@maxking: I've created a `good first issue` label and marked #179, #191, and #255 as inaugural beginner tickets. Feel free to take a look, and thanks for the interest in contributing 😇
